addons/docs/jest-transform-mdx.js

```diff
@@ -29,6 +29,9 @@ module.exports = {
       import { mdx } from '@mdx-js/react'
       ${mdx.sync(src, { compilers, filepath: filename })}
     `;
-    return getNextTransformer(filename, config).transformSource(filename, result, instrument);
+    
+    const jsFileName = `${filename}.js`;
+    
+    return getNextTransformer(filename, config).transformSource(jsFileName, result, instrument);
```

Comment:
```suggestion
    return new ScriptTransformer(config).transformSource(jsFileName, result, instrument);
```

Reply:
we would want to delete `getNextTransformer` then right?

Reply:
yep